data_validation/cli_tools.py

```diff
@@ -168,9 +172,34 @@ def configure_arg_parser():
     _configure_find_tables(subparsers)
     _configure_raw_query(subparsers)
     _configure_beta_parser(subparsers)
+    _configure_partition_parser(subparsers)
     return parser
 
 
+def _configure_partition_parser(subparsers):
+    """Configure arguments to generate partitioned config files."""
+    partition_parser = subparsers.add_parser(
+        "generate-partitions",
+        help="Generate partitions for validation and store the Config files in a directory",
+    )
+
+    # Keep these in order to support data-validation run command for backwards-compatibility
+    partition_parser.add_argument("--type", "-t", help="Type of Validation")
+    partition_parser.add_argument(
+        "--result-handler-config", "-rc", help="Result handler config details"
+    )
+
+    partition_subparsers = partition_parser.add_subparsers(
+        dest="validate_cmd",
+        help="Type of Validation.",
+    )
+
+    row_partition_parser = partition_subparsers.add_parser(
```

Comment:
I think we should only support row validations. Column validations aggregate results so memory isn't an issue (i.e count * or sum col_a will always return one row as a result, but row validations return multiple).

We can get rid of this subparser: `data-validation generate-table-partitions -sc bq -tc bq ...`

Reply:
Done